### PR TITLE
Fix XSS vulnerability for manual journal entries

### PR DESCRIPTION
**Pull request by tarnap**
Resolves: https://github.com/4teamwork/opengever.ftw/issues/32

### CODE REVIEW

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@ Changelog
 2017.3.0 (unreleased)
 ---------------------
 
+- XSS: Escape html for manual journal entries comment. [tarnap]
```
Comment (deiferni) [inline docs/HISTORY.txt]: entry comments?

Comment (tarnap) [inline docs/HISTORY.txt]: @deiferni one journal has several entries and every entry has a single comment.
Proposal:
`- XSS: Escape html for manual journal entries' comment.`

Is that valid for you?

Comment (deiferni) [inline docs/HISTORY.txt]: fair enough 😄

Comment (tarnap) [inline docs/HISTORY.txt]: @deiferni done :)

**Comment by tarnap** [comment]
@elioschmutz @phgross @maethu one more approval and I'll merge this PR

**Comment by tarnap** [comment]
Merci :+1:

**Pull request merged**